lib/PaymentMethod.php

```diff
@@ -7,16 +7,16 @@
  *
  * @property string $id
  * @property string $object
- * @property mixed|null $au_becs_debit
- * @property mixed $billing_details
- * @property mixed $card
- * @property mixed $card_present
+ * @property \Stripe\StripeObject $au_becs_debit
+ * @property \Stripe\StripeObject $billing_details
+ * @property \Stripe\Card $card
```

Comment:
AFAIK this is totally incorrect. a PaymentMethod's `card` is just a hash and not a Card object